packages/ui/src/components/va-counter/validateCounter.ts

```diff
@@ -0,0 +1,31 @@
+import { warn } from '../../services/utils'
+
+export const validateCounter = (value: string | number, step: number, min?: number, max?: number) => {
+  const val = Number(value)
+  if (!Number.isNaN(val)) {
+    if (min && max) {
+      if (max < min) {
+        warn(`The maximum value (${max}) can not be less than the minimum value (${min}).`)
+      }
+
+      if (min > max) {
+        warn(`The minimum value (${min}) can not be greater than the maximum value (${max}).`)
+      }
```

Comment:
```suggestion
      if (max < min) {
        warn(`The maximum value (${max}) can not be less than the minimum value (${min}).`)
      }
```
one of these warnings is enough